### PR TITLE
Update Translations

### PR DESCRIPTION
**Pull request by PanRadaCloudBest**

### CODE REVIEW

File: loc/en.js [hunk 1]
```diff
@@ -363,9 +372,9 @@ module.exports = {
       type: 'Transaction type',
       watchOnlyError: 'Watch only wallets cannot send transactions',
     },
-    warning: 'Warning: ',
+    warning: 'Warning:',
     warningGeneral:
-      'Please be aware that in the process of using the Secure Transaction feature, a part of the funds left in your wallet might be blocked. This is a normal procedure linked to UTXO and the blockchain parameters of the Bitcoin Vault wallet. Your funds will be unblocked once the transaction is verified (after approximately 24 hrs) or canceled (within 24 hrs).',
+      'Warning: Please be aware that in the process of using the Secure Transaction feature, a part of the funds left in your wallet might be blocked. This is a normal procedure linked to UTXO and the blockchain parameters of the Bitcoin Vault wallet. Your funds will be unblocked once the transaction is verified (after approximately 24 hrs) or canceled (within 24 hrs).',
```
Comment (KedziaPawel) [inline loc/en.js:377]: recently we deleted "Warning" from here

Comment (PanRadaCloudBest) [reply loc/en.js:377]: pls check

Comment (marek-siemieniuk-morawski) [reply loc/en.js:377]: I can confirm we removed the word "Warning: " from line 377 in another PR recently. Have you pulled the latest develop?

Comment (PanRadaCloudBest) [reply loc/en.js:377]: done

File: loc/en.js [hunk 2]
```diff
@@ -363,7 +372,7 @@ module.exports = {
       type: 'Transaction type',
       watchOnlyError: 'Watch only wallets cannot send transactions',
     },
-    warning: 'Warning: ',
```
Comment (marek-siemieniuk-morawski) [inline loc/en.js:366]: Sebastian, removing whitespace here is also a mistake. Please add it back

**Review by marek-siemieniuk-morawski — APPROVED** [review]

**Pull request merged**